prowler/config/config.py

```diff
@@ -34,7 +35,14 @@
         and file.name.endswith(".json")
         and available_compliance_frameworks.append(file.name.removesuffix(".json"))
     ]
-
+with os.scandir(compliance_gcp_dir) as files:
+    files = [
+        file.name
+        for file in files
+        if file.is_file()
+        and file.name.endswith(".json")
+        and available_compliance_frameworks.append(file.name.removesuffix(".json"))
+    ]
```

Comment:
Could you please reutilise the above code to iterate through the compliance directories?